token/js/client/token.js

```diff
@@ -2103,4 +2146,45 @@ export class Token {
       data,
     });
   }
+
+  static async getAssociatedTokenAddress(
+    associatedProgramId: PublicKey,
+    programId: PublicKey,
+    owner: PublicKey,
+    mint: PublicKey,
+  ): Promise<PublicKey> {
+    return (
+      await PublicKey.findProgramAddress(
+        [owner.toBuffer(), programId.toBuffer(), mint.toBuffer()],
+        associatedProgramId,
+      )
+    )[0];
+  }
+
+  static createAssociatedTokenAccountInstruction(
+    associatedProgramId: PublicKey,
+    programId: PublicKey,
+    payer: PublicKey,
+    owner: PublicKey,
+    mint: PublicKey,
+    associatedAccount: PublicKey,
```

Comment:
@jstarry strictly speaking, this is redundant, but I rather wanted instruction helper functions to be `not async` for consistency. Alternatively, should I rename to like `createAssociatedTokenAccountInstructionAsync()`? ;)

Reply:
Haha, I like the way you designed it